examples/dt/uni04delta/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  #  - ../../../va/hci/edpm-post-ceph/nodeset
+  - ../../../dt/uni04delta/edpm/nodeset
+
+resources:
+  - control-plane/nncp/values.yaml
+  - control-plane/service-values.yaml
+  - values.yaml
```

Comment:
[Here](https://github.com/openstack-k8s-operators/architecture/actions/runs/9028525852/job/24811285904?pr=191#step:4:18) is complaining about values.yaml missing

Reply:
Fixed